### DISCUSSION TITLE
How to change a file name when using --add-data?

### DISCUSSION THREAD

**Discussion by GabrielLacoste** [pyinstaller/pyinstaller#6821] [PyInstaller] [answered]
I want to rename a file when it copy using the --add-data parameter:
`pyinstaller --add-data 'src/__version__py:./version.txt' myscript.py`

Right now this create a directory name version.txt with the file __version__.py inside, but I want is the file rename to version.txt.
How can I do that?

Thanks for your help.

**Comment by rokm** [comment, marked as answer]
You cannot - the current implementation does not support renaming files as they are collected.
